examples/configs/pairwise_toilets.yaml

```diff
@@ -0,0 +1,83 @@
+task:
```

Comment:
I think this is bad idea to not have validation, so how you will know whether the work your code or not? Because for training set it's obviously will learn. We must be confidence that our code work well. If you can't create validation, just find another dataset.